content/docs/operating/integrations.md

```diff
@@ -50,6 +50,7 @@ For notification mechanisms not natively supported by the Alertmanager, the
   * [Phabricator / Maniphest](https://github.com/knyar/phalerts)
   * [SMS](https://github.com/messagebird/sachet): supports [multiple providers](https://github.com/messagebird/sachet/blob/master/examples/config.yaml)
   * [Telegram bot](https://github.com/inCaller/prometheus_bot)
+  * [prom2teams](https://github.com/idealista/prom2teams)
```

Comment:
It's not obvious to me what this talks to, "teams" is a very generic term. Can you clarify?

Reply:
Added short description to clarify :)

Reply:
That's a bit too wordy now, we just need to get the phrase "Microsoft Teams" in there

Reply:
I've removed some words to make it less verbose